cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -70,8 +71,10 @@ func encodeConfig(cfg *config.KubeSchedulerConfiguration) (*bytes.Buffer, error)
 		encoder = scheme.Codecs.EncoderForVersion(info.Serializer, configv1beta2.SchemeGroupVersion)
 	case configv1beta3.SchemeGroupVersion.String():
 		encoder = scheme.Codecs.EncoderForVersion(info.Serializer, configv1beta3.SchemeGroupVersion)
+	case configv1.SchemeGroupVersion.String():
+		encoder = scheme.Codecs.EncoderForVersion(info.Serializer, configv1.SchemeGroupVersion)
 	default:
-		encoder = scheme.Codecs.EncoderForVersion(info.Serializer, configv1beta3.SchemeGroupVersion)
+		encoder = scheme.Codecs.EncoderForVersion(info.Serializer, configv1.SchemeGroupVersion)
```

Comment:
In https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api#proposal, it says that we would mark v1beta2 as deprecated. It would be useful to emit a log line for that.

And of course, we should update the documentation.

Reply:
Yes, I didn't forget that, I was plan to mark v1beta2 as deprecated in another PR, this PR only focuses on the graduation.